### PR TITLE
pythonPackages.boto3: 1.9.96 -> 1.9.152 | botocore: 1.12.96 -> 1.12.152 | awscli:1.16.106 -> 1.16.162

### PR DESCRIPTION
**Pull request by PsyanticY**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [X] other Linux distributions (ubuntu)
- [X] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by AmineChikhaoui** [comment]
@GrahamcOfBorg build pythonPackages.boto3 pythonPackages.botocore awscli

File: pkgs/development/python-modules/boto3/default.nix [hunk 1]
```diff
@@ -13,11 +13,11 @@
 
 buildPythonPackage rec {
   pname =  "boto3";
-  version = "1.9.96"; # N.B: if you change this, change botocore too
+  version = "1.9.152"; # N.B: if you change this, change botocore too
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "c103241394d396ee08548b03d5d1f0f89a7ad1dfa7ccca88a47131f329cca093";
+    sha256 = "14x0rhlz6cbilj4cmk57vz8pm6pqyb8xa0wbah4fx5nkjh9j80y1";
```
Comment (jonringer) [inline pkgs/development/python-modules/boto3/default.nix]: this is the wrong hash,

```
diff --git a/pkgs/development/python-modules/boto3/default.nix b/pkgs/development/python-modules/boto3/default.nix
index 0134bee194a..c5d51e56fc6 100644
--- a/pkgs/development/python-modules/boto3/default.nix
+++ b/pkgs/development/python-modules/boto3/default.nix
@@ -17,7 +17,7 @@ buildPythonPackage rec {

   src = fetchPypi {
     inherit pname version;
-    sha256 = "14x0rhlz6cbilj4cmk57vz8pm6pqyb8xa0wbah4fx5nkjh9j80y1";
+    sha256 = "0qspp97rddshjfqmmnqbvrszr66wq0ag05j8lcfsb9rj6ymz5s5s";
   };

   propagatedBuildInputs = [ botocore jmespath s3transfer ] ++ lib.optionals (!isPy3k) [ futures ];
```

Comment (PsyanticY) [reply pkgs/development/python-modules/boto3/default.nix]: Fixed, seems like i used some other version hash when i was figuring out the relation between the cli boto3 and botocore

**Comment by FRidh** [comment]
@GrahamcOfBorg build awscli

**Pull request merged**